hummingbot/connector/derivative/coinflex_perpetual/coinflex_perpetual_order_book.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from typing import Dict, Optional
+
+from hummingbot.core.data_type.common import TradeType
+from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book_message import OrderBookMessage, OrderBookMessageType
+from hummingbot.logger import HummingbotLogger
+
+
+class CoinflexPerpetualOrderBook(OrderBook):
```

Comment:
In the latest versions of Binance we are avoiding the creation of new OrderBook classes for new connectors. Since the utility methods to create the snapshot messages, the diff messages and the trade messages do not provide many advantages and are only used in the order book data source, we should have the messages creation logic in that class and remove the specific order book class